wbia/dtool/depcache_table.py

```diff
@@ -138,7 +138,10 @@ def ensure_config_table(db):
     """ SQL definition of configuration table. """
     config_addtable_kw = ut.odict(
         [
-            ('tablename', CONFIG_TABLE,),
+            (
+                'tablename',
+                CONFIG_TABLE,
+            ),
```

Comment:
@bluemellophone You're making lots of whitespace changes in this commit.  It's adding a lot of extra lines and doesn't make the code more readable.  Are you running pre-commit with the options that we have?  I don't understand...

Reply:
```
(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ pre-commit --version
pre-commit 2.7.1

(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ pip freeze | grep black
black==20.8b1

(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ pip freeze | grep brunette
brunette==0.1.7

(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ cat pyproject.toml
[build-system]
requires = ["setuptools>=42", "wheel", "setuptools_scm[toml]>=3.4", "scikit-build", "cmake!=3.18.2", "ninja"]

(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ cat setup.cfg
[flake8]
exclude = .git,.env
# This section configures `flake8`, the python linting utility.
# See also https://flake8.pycqa.org/en/latest/user/configuration.html
ignore = D100,D101,D102,D103,D105,D200,D205,D210,D400,D401,D403,E203,E501,N802,N803,N804,N805,N806,W503

# D100 - Missing docstring in public module
# D101 - Missing docstring in public class
# D102 - Missing docstring in public method
# D103 - Missing docstring in public function
# D105 - Missing docstring in public package
# D200 - One-line docstring should fit on one line with quotes
# D205 - 1 blank line required between summary line and description
# D210 - No whitespaces allowed surrounding docstring text
# D400 - First line should end with a period
# D401 - First line should be in imperative mood
# D403 - First word of the first line should be properly capitalized
# D* codes come from pydocstyle which is integrated into flake8 via flake8-docstring.
# See also http://www.pydocstyle.org/en/5.0.1/error_codes.html

# E203 - whitespace before ':'
# E501 - line too long (82 > 79 characters)
# E* codes come from pycodestyle which is integrated with flake8
# See also https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

# N802 - function name should be lowercase
# N803 - argument name should be lowercase
# N804 - first argument of a classmethod should be named ‘cls’
# N805 - first argument of a method should be named ‘self’
# N806 - variable in function should be lowercase
# N* codes come from pep8-naming, which integrates with flake8
# See also https://github.com/PyCQA/pep8-naming#error-codes

# W503 - line break before binary operator
# W* codes come from pycodestyle which is integrated with flake8
# See also https://pycodestyle.pycqa.org/en/latest/intro.html#error-codes

[tool:brunette]
line-length = 90
verbose = false
# skip-string-normalization = true
single-quotes = true

[tool:pytest]
minversion = 5.4
addopts = -p no:doctest --xdoctest --xdoctest-style=google -r fEs
testpaths =
    wbia
filterwarnings =
    default
    ignore:.*No cfgstr given in Cacher constructor or call.*:Warning
    ignore:.*Define the __nice__ method for.*:Warning

[metadata]
# This includes the license file(s) in the wheel.
license_files = LICENSE.txt

(wildme3.7) [wildme] jason.parham in ~/code/wildbook-ia on karenc/build-process-merge*
⚡ cat .pre-commit-config.yaml
# See https://pre-commit.com for more information
# See https://pre-commit.com/hooks.html for more hooks
exclude: |
  (?x)(
      ^docs/ |
      ^_docs/_build/html/ |
      wbia/web/static/
  )
repos:
  # - repo: https://github.com/psf/black
  #   rev: stable
  #   hooks:
  #     - id: black
  #       language_version: python3.6
  - repo: local
    hooks:
      - id: brunette
        name: brunette
        description: Run Brunette on Python code (fork of Black).
        entry: brunette --config=setup.cfg
        language: system
        types: [python]
  - repo: https://gitlab.com/pycqa/flake8
    rev: 3.8.3
    hooks:
      - id: flake8
  - repo: https://github.com/pre-commit/pre-commit-hooks
    rev: v3.1.0
    hooks:
      - id: requirements-txt-fixer
      - id: check-toml
      # - id: check-json
      - id: check-yaml
        exclude: |
          (?x)(
              ^wbia/detecttools/pypascalmarkup/template_.*yml
          )
      # - id: check-ast
      - id: check-executables-have-shebangs
      # - id: check-docstring-first
      - id: detect-aws-credentials
      - id: detect-private-key
      - id: double-quote-string-fixer
      - id: trailing-whitespace
      - id: mixed-line-ending
      - id: end-of-file-fixer
      - id: fix-encoding-pragma
```

Reply:
When I run the linter on the entire codebase, I use the command: `pre-commit run --all-files`

Reply:
This is what I'm running

```
(py3) karen@yukihira:~/src/wildbook/wildbook-ia$ python --version
Python 3.8.5
(py3) karen@yukihira:~/src/wildbook/wildbook-ia$ pip --version
pip 20.1.1 from /home/karen/src/wildbook/py3/lib/python3.8/site-packages/pip (python 3.8)
(py3) karen@yukihira:~/src/wildbook/wildbook-ia$ pre-commit --version
pre-commit 2.6.0
(py3) karen@yukihira:~/src/wildbook/wildbook-ia$ pip freeze | grep black
black==19.10b0
(py3) karen@yukihira:~/src/wildbook/wildbook-ia$ pip freeze | grep brunette
brunette==0.1.7
```

Reply:
From black 20.8b0's changelog:

```
re-implemented support for explicit trailing commas: now it works
consistently within any bracket pair, including nested structures (#1288
and duplicates)
```

We have lots of trailing commas throughout our source code and it causes
changes like this:

```diff
-    print('Brambox: %r, %r' % (brambox.__version__, brambox.__file__,))
+    print(
+        'Brambox: %r, %r'
+        % (
+            brambox.__version__,
+            brambox.__file__,
+        )
+    )
```

which adds lots of lines for no reason.  There are more extreme cases that make
the code less readable after black reformats the code.  The only way to avoid
this is to remove trailing commas.

^ latest commit fixes this